### PR TITLE
Add module documentation

### PR DESCRIPTION
**Pull request by Glowbal**
**When merged this pull request will:**
- Adds documentation explaining the module pbo 
- Adds documentation explaining how to use the module framework

### CODE REVIEW

File: docs/wiki/feature/modules.md [hunk 1]
```diff
@@ -8,8 +8,14 @@ parent: wiki
 
 ## 1. Overview
 
-This is taking care of module initialization **DO NOT REMOVE**.
+This is taking care of module initialization. It lets us ensure that modules placed in the editor run when they are supposed to.
+
+ **DO NOT REMOVE THIS PBO**. It is depended upon by a large collection of ACE3 components / PBOs. Removing it will break their functionality.
 
 ## 2. Dependencies
```
Comment (jonpas) [inline docs/wiki/feature/modules.md]: This should always be the last point.

**Comment by jonpas** [comment]
I'd add a link zo BI's module framework wiki, so people can read on what properties are for what.

File: docs/wiki/feature/modules.md (hunk 1 above)
Comment (jonpas) [inline docs/wiki/feature/modules.md]: Oh, maybe not, I should really check our guidelines first...

**Comment by Glowbal** [comment]
@jonpas done

File: docs/wiki/feature/modules.md [hunk 2]
```diff
@@ -14,8 +14,14 @@ version:
 
 ## 1. Overview
 
-This is taking care of module initialization **DO NOT REMOVE**.
+This is taking care of module initialization. It lets us ensure that modules placed in the editor run when they are supposed to.
+
+ **DO NOT REMOVE THIS PBO**. It is depended upon by a large collection of ACE3 components / PBOs. Removing it will break their functionality.
```
Comment (jonpas) [inline docs/wiki/feature/modules.md]: In "main" and "common" docs we wrote it under dependencies list, like that:

> _Note: The Modules module is required by nearly all other modules. Do NOT remove it!_

**Comment by Glowbal** [comment]
@jonpas done.

**Comment by jonpas** [comment]
Looks good to me, but if we want to keep it consistent then it has to be under dependencies. [Example](http://ace3mod.com/wiki/feature/common.html)

**Pull request merged**